README.md

```diff
@@ -87,14 +87,42 @@ You can join the team by posting a comment to
 
 ### Project Maintainers
 
-|                                                  |                                                  |                                                            |
-| :----------------------------------------------: | :----------------------------------------------: | :--------------------------------------------------------: |
-|                  Biniam Admikew                  |                    Diana Lau                     |                         Janny Hou                          |
-|    [![b-admike]](http://github.com/b-admike)     |      [![dhmlau]](http://github.com/dhmlau)       |         [![jannyhou]](http://github.com/jannyHou)          |
-|                    Hage Yaapa                    |                 Nora Abdelgadir                  |                       Mario Estrada                        |
-| [![hacksparrow]](https://github.com/hacksparrow) | [![nabdelgadir]](https://github.com/nabdelgadir) | [![marioestradarosa]](https://github.com/marioestradarosa) |
-|                   Hugo Da Roit                   |                 Dominique Emond                  |                         Agnes Lin                          |
-|        [![yaty]](https://github.com/yaty)        |     [![emonddr]](https://github.com/emonddr)     |         [![agnes512]](https://github.com/agnes512)         |
+<table>
```

Comment:
Not good idea to place HTML here.  Even the documentation is MarkDown based. I believe the GitHub flavored parser don't support table without headers anyway.

Reply:
@marioestradarosa 

>  I believe the GitHub flavored parser don't support table without headers anyway.

AFAICT, GitHub's markdown parser handles HTML very well. Here is a screenshot from https://github.com/strongloop/loopback-next/blob/796e87898d929dfced7a1de215b854d1a01f3b88/README.md:

<img width="488" alt="Screen Shot 2019-06-24 at 09 57 04" src="https://user-images.githubusercontent.com/1140553/60001329-898c1380-9666-11e9-8e1b-af37b859ae48.png">

> Not good idea to place HTML here. Even the documentation is MarkDown based.

Could you please go into more details why you think it's not a good idea to use HTML?

Are you concerned about inconsistency - some places using Markdown tables and other places using raw HTML? Anything else?

Personally, I am fine to use raw HTML in places where it make sense, as an exception to the general rule of preferring Markdown tables everywhere else.

@strongloop/loopback-maintainers thoughts?

Reply:
I think my question in mind is it seems to be rendering fine (at least to me) currently in https://github.com/strongloop/loopback-next#team, why do we need to change it?  Or maybe I'm not experiencing the problem @kerolloz is seeing.

Reply:
<img width="504" alt="Screen Shot 2019-06-24 at 11 11 41 AM" src="https://user-images.githubusercontent.com/42985749/60030423-eee50200-9670-11e9-8568-3aadfe449859.png">
I see it on my end too.